src/cli/conio.hpp

```diff
@@ -41,6 +45,7 @@ extern "C"
      * @brief Gets user input lin
      * @param buffer buffer to write too
      * @param buflen length of buffer
+     * @return int value of user inputs in the buffer array excluding '\b'
```

Comment:
What does the `\b` character do/mean?  More specifically, which key on the keyboard causes this character to be emitted?